### PR TITLE
support opensuse 42.3 for nsupdate_web

### PR DESCRIPTION
**Pull request by kshtsk**

### CODE REVIEW

**Comment by kshtsk** [comment]
@zmc  @djgalloway  could you please review

File: roles/nsupdate_web/tasks/main.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 - name: Including major version specific variables.
   include_vars: "{{ item }}"
   with_first_found:
-    - "{{ ansible_distribution | lower }}_{{ ansible_distribution_major_version }}.yml"
+    - "{{ ansible_distribution | lower | replace(' ', '_') }}_{{ ansible_distribution_major_version }}.yml"
```
Comment (djgalloway) [inline roles/nsupdate_web/tasks/main.yml]: What is the `ansible_distribution` var for Leap 42.3?  Just trying to understand what this is for.

Comment (kshtsk) [reply roles/nsupdate_web/tasks/main.yml]: @djgalloway "openSUSE Leap"

Comment (djgalloway) [reply roles/nsupdate_web/tasks/main.yml]: Ah, okay.  Thanks.

**Pull request merged**